code/mission/missionmessage.cpp

```diff
@@ -1449,6 +1449,11 @@ void message_queue_process()
 			if ((Playing_messages[i].wave.isValid()) && (snd_time_remaining(Playing_messages[i].wave) > 250))
 				wave_done = 0;
 
+			// Don't kill paused messages
+			if ((Playing_messages[i].wave.isValid()) && snd_is_paused(Playing_messages[i].wave)) {
```

Comment:
Nice fix, sorry to be nitpicky but if you feel so inclined this could be an or check with the above if, though I reckon you kept them separate for readability.

Reply:
I did that originally but yeah I thought this was easier to read and there was already precedent for multiple statements that set `wave_done = 0`

Reply:
I'm good with it as is, if someone else really wants to have it be an OR then they can let us know.